docker/Dockerfile-workers

```diff
@@ -25,7 +25,7 @@ RUN dnf -y install \
     systemd-devel \
     && dnf clean all
 
-ENV OPM_SRC="/src"
+ENV OPM_SRC="/src/operator-registry"
```

Comment:
I guess it's not needed it just seemed strange to clone the operator-registry repo in src/ instead of in a dedicated nested folder. I can revert that if you prefer.

Reply:
Nah, it's fine. I was just curious. A separate commit might be nice :)